portfolio/src/main/webapp/index.html

```diff
@@ -5,34 +5,67 @@
   <title>Fiza Goyal</title>
   <link rel="stylesheet" href="style.css">
   <script src="script.js"></script>
+
+  <link href="https://fonts.googleapis.com/css2?family=Slabo+27px&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
 </head>
 <body>
-  <div class="header" id="stickyHeader">
+  
+  <!--------------HEADER-------------->
+  <header>
     <h2 class = "title">Fiza Goyal</h2>
     
     <!-- The navigation menu -->
-    <div class="navbar">
-      <a class="active" href="#about">About</a>
-      <a href="#experience">Experience</a>
-      <a href="#">Projects</a>
-      <a href="#">Contact</a>
-  </div>
-  </div>
+    <nav class = "top-navbar">
+      <ul>
+        <a class="active" href="#about"><li>Home</li></a>
+        <a href="#about"><li>About</li></a>
```

Comment:
It seems perhaps a little confusing to have multiple links going to the same place. I would use either "Home" or "About" as the navigation element.

Reply:
Thanks for catching that! I was originally going to have two separate sections for Home and About and just forgot to get rid of the link.